src/Symfony/Component/DependencyInjection/ServiceProviderInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection;
+
+use Psr\Container\ContainerInterface;
+
+/**
+ * A ServiceProviderInterface exposes identifiers and types of services provided by a container.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ * @author Mateusz Sip <mateusz.sip@gmail.com>
+ */
+interface ServiceProviderInterface extends ContainerInterface
+{
+    /**
+     * Returns an associative array of service types keyed by names provided by this object.
+     *
+     * Examples:
+     *
+     *  * array('logger' => 'Psr\Log\LoggerInterface') means the object provides service implementing Psr\Log\LoggerInterface
+     *    under "logger" name
+     *  * array('foo' => '?') means that object provides service of unknown type under 'foo' name
+     *  * array('bar' => '?Bar\Baz') means that object provides service implementing Bar\Baz or null
```

Comment:
missing `under "bar" name` for consistency

Reply:
thanks, fixed.